docs/source/deployment/knative.rst

```diff
@@ -1,41 +1,44 @@
-Deploying to Knative
+Deploying to KNative
 ====================
 
 Knative is kubernetes based platform to deploy and manage serverless workloads. It is a
 solution for deploying ML workload that requires more computing power that abstracts away
 infrastructure management and without worry about vendor lock.
 
+This guide demonstrates how to serve a scikit-learn based iris classifier model with
+BentoML on a KNative cluster. The same deployment steps are also applicable for models
+trained with other machine learning frameworks, see more BentoML examples :doc:`here <../examples>`.
 
 Prerequisites
 -------------
 
-* kubernetes cluster version 1.15 or newer.
+* A kubernetes installed cluster. `minikube` is used in this guide.
```

Comment:
* A Kubernetes cluster
    * `minikube` is the recommended way to run Kubernetes locally: https://kubernetes.io/docs/tasks/tools/install-minikube/
    * Kubernetes guide: https://kubernetes.io/docs/setup/